_data/roles.yml

```diff
@@ -1,3 +1,35 @@
+- name: Code of Conduct Committee Member
+  open: true
+  description: |
+    The Code of Conduct (COC) Committee member is responsible for ensuring the OBO Code of Conduct is upheld.
+    They are, in particular, available for anyone in the community to be contacted individually or as a group.
+  commitment: 0-6 hours per month
+  requirements:
+    - being able to deal with sensitive interpersonal issues discreetly
+    - being willing to mediate between conflicting viewpoints
+    - being highly available when issues arise
+    - fully embracing the OBO Code of Conduct
+  responsibilities:
+    - Being available to receive COC-related issues via email from anyone in the community, and dealing with them discreetly (not sharing any part of the communication without permission)
+    - Developing strategies for dealing with COC-related issues which are often socio-technologically complex
+    - Ensuring that the OBO Code of Conduct is upheld around matters directly related to the OBO Foundry
+    - (Other responsibilities are yet to be defined for this role)
+  people:
+    - github: addiehl
+      name: Alexander Diehl
+      orcid: 0000-0001-9990-8331
+      status: support
+      start: 2023-05-16
```

Comment:
```suggestion
      start: "2023-05-16"
```

Reply:
I can change the code handling of these in a different PR, but just accept these changes to make the PR pass the tests

@matentzn

Reply:
also please double check the start dates, they are not consistent. some say 2023-05-16 and some say 2023-06-16.

Reply:
I used the date whatever person declared their wish to participate, but I don't think the exact date matters that much.. Its fine like you suggest.